src/icons/file/README.md

```diff
@@ -27,6 +27,10 @@ const icons = [
         name: 'FileBoxNote32',
         component: require('../../icon/content/FileBoxNote32').default,
     },
+    {
+        name: 'FileCanvas32',
```

Comment:
Should this be `FileBoxCanvas32`? Should the file itself also be renamed?

Reply:
Do we need to update the design asset to follow the new naming convention?

Reply:
Yeah this area I'm a little conflicted so it's more of a compromise for me.

Ideally, we should be using the same names across the different repos: design-assets, box-ui-elements, and the monolith. But because they're owned/maintained by different groups, it seems unlikely we'll be able to stay in sync.

With the exception of `FileBoxNote32`, I think the design team has been pretty consistent with their naming structure. For example, I'm seeing the following pattern: `File[product][size]`

- `FileExcel32`
- `FileSheets32`
- `FileKeynote32`
- `FilePhotoshop32`

In these cases, the file names do not include the company name which makes me lean towards `FileCanvas32` being correct here from the design team's perspective. Otherwise, we would need to rename all the files in the design-assets repo to follow the `File[brand][product][size]` pattern.

When these files are imported into BUIE is where we've seemed to stray from a standard. I think adding the company/brand in the name adds more clarity about the icon but we've only done it with the Google icons so maybe it was correct the other way? The Apple, Adobe, and Microsoft icons only have the product name.

The inconsistencies seem to stem from this table: https://github.com/box/box-ui-elements/blob/master/src/icons/item-icon/ItemIcon.tsx#L39-L73.
Because of the usages of these key values in the monolith, we can't (easily) rename the keys so it looks like we've been mirroring the variable names after them.

E.g. we currently have
```
'excel-spreadsheet': IconFileExcelSpreadsheet,
'google-slides': IconFileGoogleSlides,
```
but it should probably be
```
'excel-spreadsheet': IconFileExcel,
'google-slides': IconFileSlides,
```

I'm okay with merging this as-is and then I can address all the inconsistencies from the BUIE side in a future PR.